mappings/net/minecraft/item/Item.mapping

```diff
@@ -59,6 +63,7 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_7844 getAttributeModifiers (Lnet/minecraft/class_1304;)Lcom/google/common/collect/Multimap;
 		ARG 1 slot
 	METHOD method_7846 isDamageable ()Z
+		COMMENT @return whether an ItemStack with this Item can loose durability.
```

Comment:
```suggestion
		COMMENT @return whether this Item can lose durability.
```
The mention of `ItemStack` here is unique from other similar docs, so might be worth excluding it here